src/script/lua_api/l_itemstackmeta.h

```diff
@@ -58,12 +58,13 @@ class ItemStackMetaRef : public MetaDataRef
 	// garbage collector
 	static int gc_object(lua_State *L);
 public:
-	ItemStackMetaRef(ItemStack *istack): istack(istack) {}
-	~ItemStackMetaRef() = default;
+	// takes a reference
+	ItemStackMetaRef(LuaItemStack *istack);
+	~ItemStackMetaRef();
```

Comment:
Also disable class copy here, or use a `irr_ptr`-like smart pointer.
(Otherwise, there's a double-free after copy or move.)

Reply:
Doesn't look resolved.

Reply:
Whoops, I disabled copying for the wrong class.